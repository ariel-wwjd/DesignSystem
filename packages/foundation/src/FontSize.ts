const fontSizes: {
  xs: 'sx',
  sm: 'sm',
  base: 'base',
  lg: 'lg',
  xl: 'xl',
} = {
  xs: 'sx',
  sm: 'sm',
  base: 'base',
  lg: 'lg',
  xl: 'xl',
};

export default Object.freeze(fontSizes);
