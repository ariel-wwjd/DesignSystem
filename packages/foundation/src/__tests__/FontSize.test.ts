import FontSize from '../FontSize';

test('smapshot of font sizes ', () => {
  expect(FontSize).toMatchSnapshot();
});
