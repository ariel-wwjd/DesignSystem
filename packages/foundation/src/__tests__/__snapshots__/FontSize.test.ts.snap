// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smapshot of font sizes  1`] = `
{
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "sx",
}
`;
